demisto_sdk/commands/content_graph/neo4j_service.py

```diff
@@ -102,17 +137,16 @@ def start(use_docker: bool = True):
             name="neo4j-content",
             ports={"7474/tcp": 7474, "7687/tcp": 7687, "7473/tcp": 7473},
             volumes=[
-                f'{REPO_PATH / NEO4J_FOLDER / NEO4J_DATA_FOLDER}:/{NEO4J_DATA_FOLDER}',
-                f'{REPO_PATH / NEO4J_FOLDER / NEO4J_IMPORT_FOLDER}:{LOCAL_NEO4J_PATH / NEO4J_IMPORT_FOLDER}',
-                f'{REPO_PATH / NEO4J_FOLDER / NEO4J_PLUGINS_FOLDER}:/{NEO4J_PLUGINS_FOLDER}'
+                f"{REPO_PATH / NEO4J_FOLDER / NEO4J_DATA_FOLDER}:/{NEO4J_DATA_FOLDER}",
+                f"{REPO_PATH / NEO4J_FOLDER / NEO4J_IMPORT_FOLDER}:{LOCAL_NEO4J_PATH / NEO4J_IMPORT_FOLDER}",
+                f"{REPO_PATH / NEO4J_FOLDER / NEO4J_PLUGINS_FOLDER}:/{NEO4J_PLUGINS_FOLDER}",
             ],
             detach=True,
             environment={
                 "NEO4J_AUTH": f"neo4j/{NEO4J_PASSWORD}",
                 "NEO4J_apoc_export_file_enabled": "true",
                 "NEO4J_apoc_import_file_enabled": "true",
                 "NEO4J_apoc_import_file_use__neo4j__config": "true",
-                "NEO4JLABS_PLUGINS": '["apoc"]',
```

Comment:
can we somehow keep this option as a backup for users that are not connected to the internet?

Reply:
this option will download APOC from the internet anyway, so if they will not connect to the internet it will not help and they will have to do it in advance